tickit/devices/eiger/eiger_adapters.py

```diff
@@ -77,22 +67,22 @@ async def put_config(self, request: web.Request) -> web.Response:
 
         if self.device.get_state()["value"] != State.IDLE.value:
             LOGGER.warning("Eiger not initialized or is currently running.")
-            return web.json_response(serialize(SequenceComplete(7)))
+            return web.json_response(serialize([]))
         elif (
             hasattr(self.device.settings, param)
             and self.device.get_state()["value"] == State.IDLE.value
         ):
             attr = response["value"]
 
-            LOGGER.debug(f"Changing to {attr} for {param}")
+            LOGGER.debug(f"Changing to {str(attr)} for {str(param)}")
```

Comment:
It is recommended to use `!s` to perfrm the call to `__str__`. See: https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals